roles/include_components/README.md

```diff
@@ -1,15 +1,46 @@
-# include_components role
+# redhat.ocp.include_components role - create and attach DCI components to the DCI jobs
 
-The `include_components` role adds components in the DCI jobs from software resources available in the ansible controller node, for example: rpm packages, and git repositories used during the job execution.
+- [Synopsis](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#synopsis)
+- [Parameters](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#parameters)
+- [Examples](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#examples)
+
+# Synopsis
+
+The `include_components` role creates and attaches DCI components to the DCI jobs by generating them from provided resources. Possible resource inputs include RPM packages, Git repositories, and Podman images.
```

Comment:
Should be repo commits instead of podman images?

Reply:
yes, let's rename everything to GH components